enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/use-breakout-data.ts

```diff
@@ -13,10 +15,10 @@ export interface SDKBreakoutItem extends BreakoutListItem {
   replaceBreakoutColumn: (column: Lib.ColumnMetadata) => void;
 }
 
-export const useBreakoutData = ({
-  question,
-  updateQuestion,
-}: QuestionStateParams): SDKBreakoutItem[] => {
+export const useBreakoutData = (): SDKBreakoutItem[] => {
+  const { updateQuestion, ...interactiveQuestionContext } =
+    useInteractiveQuestionContext();
+  const question = interactiveQuestionContext.question as Question;
```

Comment:
let me know what you think. This is to ensure, question always has a value. This isn't the best, but the way we render this is that `useBreakoutData` is only called when `question` has a value.

Reply:
Is that why we added the `as Question` - is it to assert that we are always not null? In that case, `const question = interactiveQuestionContext.question!` is a bit more clear to me.